helm/sematic-server/templates/socketio_deployment.yaml

```diff
@@ -0,0 +1,67 @@
+{{- if .Values.deployment.socket_io.dedicated }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-socketio
+  labels:
+    {{- include "sematic-server.labels" . | nindent 4 }}
+    app.kubernetes.io/component: socket_io
+spec:
+  {{- if not .Values.deployment.autoscaling.enabled }}
+  replicas: {{ .Values.deployment.replica_count }}
```

Comment:
There should always only be 1 socketio replica

Reply:
👍